catalogue_pipeline/recorder/src/main/scala/uk/ac/wellcome/platform/recorder/models/RecorderWorkEntry.scala

```diff
@@ -0,0 +1,18 @@
+package uk.ac.wellcome.recorder.models
+
+import uk.ac.wellcome.models.Sourced
+import uk.ac.wellcome.models.work.internal.UnidentifiedWork
+
+case class RecorderWorkEntry(
```

Comment:
Why do we wrap the `UnidentifiedWork`? `UnidentifiedWork` already contains a `sourceIdentifier` field with the same information that this wrapping class is adding.

Reply:
VHS can only store classes that have the `Id` trait, and putting that on UnidentifiedWork feels a bit weird.

Reply:
Ah right. Pff. I think we could loosen up that constraint because it's already using an implicit `IdGetter` almost everywhere to retrieve the id (Actually we would have to do that when we take the VHS out). So we could make the VHS not restrict to classes that extend `Id` and create a `IdGetter` for `UnidentifiedWork` but it would probably have a very similar logic to this so probably we wouldn't gain that much (other than loosing a wrapping case class)

Reply:
Okay, so maybe drop this for now, and instead I’ll look at making VHS not require Id?

Reply:
I'll have a poke at this a bit longer, because it would be nice to get the recorder done rather than do more refactoring.

I think not having the wrapper would be nice, but it's not a massive issue.

I will add a comment explaining why we have the wrapper.